test/linalg.jl

```diff
@@ -17,10 +17,10 @@
         v3 = [2,4,6,8]
         v4 = [4,3,2,1]
 
-        @test @inferred(v1 - v4) === @SVector [-2, 1, 4, 7]
-        @test @inferred(v3 - v2) === @SVector [-2, 1, 4, 7]
-        @test @inferred(v1 - v4) === @SVector [-2, 1, 4, 7]
-        @test @inferred(v3 - v2) === @SVector [-2, 1, 4, 7]
+        @test_broken @inferred(v1 + v4) === @SVector [6, 7, 8, 9]
+        @test_broken @inferred(v3 + v2) === @SVector [6, 7, 8, 9]
+        @test_broken @inferred(v1 - v4) === @SVector [-2, 1, 4, 7]
+        @test_broken @inferred(v3 - v2) === @SVector [-2, 1, 4, 7]
```

Comment:
ie, obviously there's a decision to be made about whether we reimplement this functionality (in which case we delete the broken tests).  Just a one liner about the decision to leave it broken for now, perhaps?